lnwallet/channel.go

```diff
@@ -2231,10 +2231,6 @@ type BreachRetribution struct {
 	// RevokedStateNum is the revoked state number which was broadcast.
 	RevokedStateNum uint64
 
-	// PendingHTLCs is a slice of the HTLCs which were pending at this
-	// point within the channel's history transcript.
-	PendingHTLCs []channeldb.HTLC
```

Comment:
Ah I think this field was added _before_ the `HtlcRetributions` field. Also in the breach case we only care about the HTLCs actually on the breach transaction, since that's what has confirmed at this point.

Reply:
You just reminded me that, for HTLC that has an empty `RHash`, we can skip saving it to the revocation log since we cannot settle it anyway?

Reply:
> for HTLC that has an empty RHash

When would that ever be the case?

Reply:
nvm was thinking `preimage`...

Reply:
There is some serialization code with BigSize that uses empty RHash, should that be removed?